services/apps/src/apps/routes/domain/permission.clj

```diff
@@ -0,0 +1,18 @@
+(ns apps.routes.domain.permission
+  (:use [common-swagger-api.schema :only [describe]]
+        [schema.core :only [defschema optional-key enum]])
+  (:import [java.util UUID]))
+
+(defschema AppIdList
+  {:apps (describe [String] "A List of app identifiers")})
```

Comment:
We do have to worry about agave here. Agave app IDs won't be UUIDs. We're not going to be supporting permission listing for apps immediately, but we will be supporting them.